api/HTMLCanvasElement.json

```diff
@@ -766,10 +767,10 @@
               }
             ],
             "firefox": {
-              "version_added": "19"
+              "version_added": "18"
             },
             "firefox_android": {
-              "version_added": "4"
+              "version_added": "18"
```

Comment:
Yep, I'll trust http://mdn-bcd-collector.appspot.com/tests/api/HTMLCanvasElement/toBlob over the existing data here, false negatives aren't plausible from how the test works.